### ISSUE TITLE
add inactive flag

### ISSUE THREAD

**Issue by tarekziade** [mozilla/servicebook#36]
Add an "inactive" flag on projects

**Comment by davehunt** [comment]
I have this working locally, however `servicebook-migrate-db` didn't work for me. Removing my local SQLite DB and running `servicebook-import` worked though. Any ideas @tarekziade?

Before I open a pull request, I'd appreciate some feedback on the UI for this change. I'm currently using a warning alert on the project page, and an alert glyph with a tooltip stating "Project is not currently active" on the projects list and search results. I think we could improve this, but I'm open to suggestions. @rbillings, @karlht?

Of course, the UI changes will be against https://github.com/mozilla/servicebook-web and not this repository. The changes here are minor, and if preferred I can open a PR for them before I complete the web app changes.

## Edit project
![screenshot-2018-1-25 edit pizza-beacon](https://user-images.githubusercontent.com/122800/35393520-153d8f78-01dd-11e8-9651-c2cad8549c24.png)

## Project
![screenshot-2018-1-25 pizza-beacon](https://user-images.githubusercontent.com/122800/35393509-0f516652-01dd-11e8-91b7-44850f7299c3.png)

## Browse projects
![screenshot-2018-1-25 http localhost](https://user-images.githubusercontent.com/122800/35393553-2ec1d68e-01dd-11e8-9bfe-a5f18c134a64.png)

## Search results
![screenshot-2018-1-25 http localhost 1](https://user-images.githubusercontent.com/122800/35393563-33805650-01dd-11e8-9134-0d85aa91acb3.png)

**Comment by tarekziade** [comment]
did you pass the config file to the migrate script?

**Comment by davehunt** [comment]
@tarekziade no, I didn't know I needed to 😳 let me try again. If it works I may open a PR with the change, and continue this conversation on servicebook-web.

**Comment by davehunt** [comment]
I'm missing something @tarekziade.. if I remove the database and recreate without my changes, I still have the new field in the db. I can create a database in a unique file without my changes but when I attempt to migrate it says there's nothing to do. Do I need to clear some cache somewhere?

**Comment by davehunt** [comment]
Ah, I've found `/tmp/qa_projects.db` which I think might be being used instead of my argument...

**Comment by davehunt** [comment]
Nope, still lost.. Help me @tarekziade! 😆

**Comment by tarekziade** [comment]
Sorry I was wrong, for the migration script, the option is the sqluri itself not the configuration file.

The default path is /tmp/qa_projects.db, but when you call the scripts you can pass your own path.
e.g.

```
$ servicebook-migrate-db --sqluri /tmp/qa_projects.db
```

**Comment by rbillings** [comment]
In regards to the UI change - I like the example banner above. Other ideas include an icon/tag bubble next to the project name, before the edit button. Or a prominent tag that displays first before the other tags displaying status. Ideally the retired status would also display on the project list page, either with shading or an icon. 
Thanks for working on this!

**Comment by rbillings** [comment]
This was added and merged.
